syntax = "proto3";

package Students;

import "Student.proto";

message StudentsGroup {
	repeated Student Students = 1;
}


